### ISSUE TITLE
To update the twitter Logo.

### ISSUE THREAD

**Issue by DhairyaMajmudar** [plone/plone.org#159]
Currently, Plone is using the old Logo of Twitter which can be replaced by its new logo

![image](https://github.com/plone/plone.org/assets/124715224/7b24ee9b-398a-419b-9dca-59827466d804)

**Comment by DhairyaMajmudar** [comment]
@stevepiercy shall I open a pr for this issue.

**Comment by stevepiercy** [comment]
@DhairyaMajmudar yes. (Although personally I think we should kick Xitter to the curb and use [Mastodon](https://plone.social/@plone) instead. Perhaps your PR will kick the Marketeers in the butt to pull the plug on Xitter.)

**Comment by Fosten** [comment]
@DhairyaMajmudar TwitterSVG is imported here https://github.com/plone/plone.org/blob/12a29cd8b0cfe08b7049b6b278ce3a6b246c7630/frontend/src/components/ShareButtons/ShareButtons.jsx#L9

To replace old Bird icon, upload new X icon here
https://github.com/plone/plone.org/blob/12a29cd8b0cfe08b7049b6b278ce3a6b246c7630/frontend/src/icons/twitter-brands.svg 

@stevepiercy The delay is Hootsuite doesn't support Mastodon yet.  A move to buffer.com this Spring should better facilitate the new socials.

**Comment by DhairyaMajmudar** [comment]
> @DhairyaMajmudar TwitterSVG is imported here
> 
> https://github.com/plone/plone.org/blob/12a29cd8b0cfe08b7049b6b278ce3a6b246c7630/frontend/src/components/ShareButtons/ShareButtons.jsx#L9
> 
> To replace old Bird icon, upload new X icon https://github.com/plone/plone.org/blob/12a29cd8b0cfe08b7049b6b278ce3a6b246c7630/frontend/src/icons/twitter-brands.svg
> 
> @stevepiercy The delay is Hootsuite doesn't support Mastodon yet. A move to buffer.com this Spring should better facilitate the new socials.

Thanks for clarification

**Comment by Vedant1745** [comment]
can i contribute to this is issue

**Comment by stevepiercy** [comment]
@Vedant1745 yes, but at your own risk. It is not clear to me whether we will continue to use this social media platform, so your work might never get merged. The @plone/marketing-communications needs to make a decision before any work is done on this issue.

If you wish to go ahead, note I just closed the PR from @yashmangal112 who did not follow contributing guidelines. Please read and follow [First-time contributors](https://6.docs.plone.org/contributing/first-time.html), especially [Things not to do](https://6.docs.plone.org/contributing/first-time.html#things-not-to-do), and [Contributing to Plone](https://6.docs.plone.org/contributing/index.html).
